crates/matrix-sdk/src/account.rs

```diff
@@ -786,6 +787,40 @@ impl Account {
 
         Ok(())
     }
+
+    /// Adds the user to the account's data ignore list
+    pub(crate) async fn ignore_user(&self, user_id: &OwnedUserId) -> Result<()> {
+        let mut ignored_user_list = self.get_ignored_user_list_event_content().await?;
+        ignored_user_list.ignored_users.insert(user_id.clone(), Default::default());
```

Comment:
```suggestion
        ignored_user_list.ignored_users.insert(user_id.clone(), IgnoredUser::new());
```

Might be a little clearer, and then you can Ctrl+Click through to see details of the type.